app/src/main/java/ca/ubc/best/mint/museandroidapp/vm/FlankerViewModel.java

```diff
@@ -108,8 +137,15 @@ public void beginStage(final FlankerStage newStage) {
         completionHandler.onComplete(this);
         return;
       }
+
+      //arrows are determined at PRE_CUE stage in order for CUE to correlate to a corresponding arrow
+      this.arrowText = stimulusArray.get(stimulusIndex).asText();
+      this.cue = stimulusCueArray.get(stimulusIndex);
```

Comment:
suggestion: can these be named currentArrowText and currentCue?
Just so it's obvious that they will be changing as we go.

One alternative would be to have a method this.currentCue() which looks it up from stimulusCueArray.get(stimulusIndex), which makes it even more obvious that it might change every time it's called.